### PR TITLE
Feature/update profile

### PR DESCRIPTION
**Pull request by brunoparma88**
Update first name, last name and gender on profile

### CODE REVIEW

File: spec/request/api/v1/users/update_spec.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
```
Comment (martinjaimem) [inline spec/request/api/v1/users/update_spec.rb]:
```suggestion

```

File: spec/request/api/v1/users/update_spec.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    end
+  end
+
+  context 'with no user logged in' do
+    subject do
+      post api_v1_targets_path, params: {}, as: :json
+      response
+    end
+
+    it 'returns unauthorized' do
```
Comment (martinjaimem) [inline spec/request/api/v1/users/update_spec.rb:67]: we should also test the action didn't take place

File: spec/request/api/v1/users/update_spec.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          gender: 'male'
+        }
+      }
+    end
+
+    it 'returns success' do
+      expect(subject).to have_http_status(:success)
+    end
+
+    it 'returns the user' do
```
Comment (martinjaimem) [inline spec/request/api/v1/users/update_spec.rb:41]: can we also test the user attrs were updated?

**Review by martinjaimem — COMMENTED** [review]
Niceee

File: config/routes.rb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  resources :users, only: %i[new create]
+  resources :users, only: %i[new create update]
```
Comment (jpascual1994) [inline config/routes.rb]: Do we need these endpoints?

**Review by martinjaimem — APPROVED** [review]

**Review by jpascual1994 — APPROVED** [review]

**Pull request merged**